CONTRIBUTING.md

```diff
@@ -1,4 +1,14 @@
 ## Contribution guidlines
 
 ### Testing
+
 #### Test deploy
+
+
+* Install [now-cli](https://zeit.co/now) using `npm i -g now` and run `now` inside the project root directory. If you are using this for the first time, follow instructions to login (requires email verification).
```

Comment:
eg. This first point can be written as

- Install [now-cli](https://zeit.co/now)
`$ npm install -global now`
- Run now on root folder
`$ now`